examples/deployment/docker/log_signer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13 as build
+FROM golang:buster as build
```

Comment:
I would slightly prefer a static Go version, for better reproduceability.

Also, what is the point of using the latest one? Are we targeting any specific feature?

Reply:
It's nice to find out that upstream golang releases break our build though...

Reply:
We'll catch it locally anyway if it happens, and also when we upgrade it next time. Not knowing which Go version is run is CI (correspondingly, not being able to tell in `README` which version to use for dev), seems worse to me.

Reply:
I'm not sure having pins in has worked very well though; we had bits pinned to 1.11, others on 1.12, 1.13, and 1.15 - and we (at least I) only found that out when I tried to bump deps which depended on parts of the golang stdlib which didn't exist...

Reply:
It'd be nice if this was defined in one place, not sure if that's even possible, just wishful thinking...